linux_os/guide/system/accounts/accounts-restrictions/password_storage/no_legacy_plus_entries_etc_passwd/ansible/shared.yml

```diff
@@ -1,17 +1,16 @@
-# platform = multi_platform_fedora,Red Hat OpenShift Container Platform 4,Oracle Linux 7,Oracle Linux 8,Red Hat Enterprise Linux 7,Red Hat Enterprise Linux 8,Red Hat Virtualization 4
+# platform = multi_platform_all
 # reboot = false
 # strategy = restrict
 # complexity = low
 # disruption = medium
 
-- name: "Backup the old /etc/passwd file"
-  copy:
-    src: /etc/passwd
```

Comment:
I believe the `src` line was removed by mistake. Could you check, please?

Reply:
Fixed.